api/custom_auth/views.py

```diff
@@ -141,6 +143,17 @@ def perform_destroy(self, instance):  # type: ignore[no-untyped-def]
             )
         )
 
+    def retrieve(self, request: Request, *args: Any, **kwargs: Any) -> Response:
```

Comment:
I'm not sure this is the right context in which to update the user's `last_login`. I wonder, is there somewhere more appropriate to implement authentication side-effects?